### PR TITLE
Default iface per bridge

### PR DESCRIPTION
**Pull request by overhacked**
Adds ability to specify a `vnet_default_iface` per bridge in the `interfaces` property. Helps make configuration more expressive for the proposed netgraph PR #843.

Instead of:
```
interfaces=vnet0:bridge0,vnet1:bridge1
```
and adding host interfaces in `rc.conf`, one can specify, for example:
```
interfaces=vnet0:bridge0:em0,vnet1:bridge1:tap1
```

Make sure to follow and check these boxes before submitting a PR! Thank you.

- [x] Explain the feature
- [x] Read [CONTRIBUTING.md](https://github.com/iocage/iocage/blob/master/CONTRIBUTING.md)

### CODE REVIEW

**Review by skarekrow — DISMISSED** [review]
LGTM, thanks!

**Comment by skarekrow** [comment]
You will have to rebase this as well in addition to fixing the travis issues :+1:

File: iocage_lib/ioc_json.py [hunk 1]
```diff
@@ -2042,6 +2042,33 @@ def json_check_prop(self, key, value, conf):
...
+                                    'message':
+                                    "Interfaces must be specified as a pair "
+                                    "or a triplet that includes a host interface "
+                                    "to be on the bridge.\n"
+                                    "EXAMPLE: vnet0:bridge0,vnet1:bridge1:em0"
+                                },
+                                _callback=self.callback,
+                                silent=self.silent
+                            )
+                        if iface.count(':') == 2 and iface.split(':')[2] not in netifaces.interfaces():
```
Comment (sonicaj) [inline iocage_lib/ioc_json.py]: @skarekrow I think this check should be done repeated on starting jail as well. It is possible that the interface no longer exists when a jail is started vs created. Thoughts ?

Comment (skarekrow) [reply iocage_lib/ioc_json.py]: @sonicaj Good point!

Comment (overhacked) [reply iocage_lib/ioc_json.py]: Agreed! I'll throw that in and rebase.

Comment (sonicaj) [reply iocage_lib/ioc_json.py]: Thank you ;)

**Review by sonicaj — DISMISSED** [review]
Thank you

**Comment by skarekrow** [comment]
@overhacked All done with this one?

**Comment by overhacked** [comment]
Reasonable question. I merged it into #843, because it made combining the netgraph and if_bridge code simpler. That said, it would only take a few minutes to remove the code for this feature from that PR so you can approve and merge separately, if you’d prefer?

**Comment by skarekrow** [comment]
@overhacked Preferrably we keep these separate, we'll merge this first so you can rebase the netgraph one on that. Makes the diff smaller. Btw a different PR was merged in the meantime, so netgraph's version for config will need to be 19.

**Comment by overhacked** [comment]
Actually, I think #843 needs to be merged first, because this changes some of the interface creation functions that it refactors. It’ll be easier to rebase these changes onto the combined functions.

**Comment by skarekrow** [comment]
Sorry about that! Some git-fu backfired, can you please reopen this? Thanks!

**Pull request closed without merging**